src/components/AccessibilityGuide/ProgressiveEnhancement.mdx

```diff
@@ -20,88 +21,182 @@ import Link from "../Link/Link";
+- `HelperErrorText`
+- `Hero`
+- `HorizontalRule`
+- `Icon`
+- `Image`
+- `Label`
+- `Link`
+- `List`
+- `Logo`
+- `SimpleGrid`
+- `SkipNavigation`
+- `StatusBadge`
+- `StructuredContent`
+- `StyledList`
+- `Table`
+- `Template`
+- `Text`
+
+The following (16) use the HTML `<button>` element and are not fully functional
+without JavaScript:
+
+- `Accordion`
+- `AlphabetFilter`
+- `Banner`
+- `Button`
+- `ButtonGroup`
+- `Header`
+- `Menu`
+- `Modal`
+- `MultiSelect`
+- `MultiSelectGroup`
+- `NewsletterSignup`
+- `Notification`
+- `Pagination`
+- `SubNav`
+- `Tabs`
+- `TagSet`
+
+The following (15) components need a pattern update to work without JavaScript:
+
+- `CheckBox`
+- `CheckboxGroup`
+- `FeedbackBox`
+- `FilterBarInline`
+- `FilterBarPopup`
+- `ProgressIndicator`
+- `Radio`
+- `RadioGroup`
+- `SearchBar`
+- `Select`
+- `SkeletonLoader`
+- `Slider`
+- `TextInput`
+- `Toggle`
+- `Tooltip`
 
 The following (2) can't be fixed due to browser limitations:
 
-- `AudioPlayer`, `VideoPlayer`
+- `AudioPlayer`
+- `VideoPlayer`
 
 The following (1) depend on external libraries:
 
 - `DatePicker`
 
 ## General Recommendations
 
-In our current web world and based on the number of users that have JS disabled,
-instead of fixing all no-JS issues, we should attempt to fix what we can. This
-audit gives us a better understanding of what components do not work without JS.
-We need to document our findings and be transparent about the components that do
-not work. Based on the Engineering General Javascript standard, a no-JS fallback
-is preferable but not required.
-
-Various new pages and features that are being built have JS-heavy requirements.
-Based on the standards, we need to recommend and encourage teams to add a
-noscript fallback message that instructs patrons to enable JS to use the site.
-
-We recommend rendering a simple component with the patron message to use as the
-fallback noscript solution. This would not be a React component but rather an
-HTML/CSS snippet.
+Based on the [Engineering General JavaScript standard](https://github.com/NYPL/engineering-general/blob/main/standards/javscript.md),
+a no-JavaScript fallback is preferable but not required. Our recommendation is
+to fix what we can. This audit gives us a better understanding of what components
+currently do not work without JavasSript, and which ones we have the opportunity
+to change (with time and resources).
+
+Various new pages and features being built for NYPL have JavaScript-heavy
+requirements. We recommend and encourage teams to add a noscript fallback
+message. This would be an HTML/CSS snippet (not React) that instructs patrons
+to enable Javascript to use the site.
+
+## noscript Fallback Example
+
+TBD
+
+<Source
+  language="html"
+  code={`
+<noscript>
+  <aside className="noscript-banner">
+    <h2>JavaScript is disabled</h2>
+    <div>Some features may not work as expected.</div>
+  </aside>
+  <style>
+    .noscript-banner {
+      background: #f9e08e;
```

Comment:
You should be able to use CSS vars without javascript enabled. Or is this necessary because the Chakra CSS variables are set using javascript?

Reply:
This makes it agnostic to where it's placed. This snippet could be added to a third-party app that doesn't use the DS, so the CSS var won't be available.